collectors/mmrepquota.go

```diff
@@ -29,10 +30,12 @@ import (
 
 var (
 	configMmrepquotaFilesystems = kingpin.Flag("collector.mmrepquota.filesystems", "Filesystems to query with mmrepquota, comma separated. Defaults to all filesystems.").Default("").String()
+	configMmrepquotaTypes       = kingpin.Flag("collector.mmrepquota.quotatypes", "Quota Types to query with mmrepquota, Default to fileset only").Default("j").String()
```

Comment:
I think it would be more intuitive if the values were like `fileset,user,group` or `fileset,user`, etc.  Then do `strings.Split(*configMmrepquotaTypes, ",")` when you go to iterate over them.  Could create static map like:

```
quoteTypeMap = map[string]string{
  "user": "u",
  "group": "g",
  "fileset": "j",
}
```